_articles/importing/import-from-1password.md

```diff
@@ -7,47 +7,61 @@ popular: false
 tags: [import, 1password]
 ---
 
-Importing your data from 1Password into Bitwarden is easy. 1Password has two versions of their application: 1Password 4 and 1Password 6 and 7. Depending on which operating system and version of 1Password and you are using, follow the proper steps below.
+To migrate from 1Password to Bitwarden, export your data from 1Password and use the Bitwarden Import Tool to quickly fill your Vault with logins.
 
-## Export Your 1Password 1pif Logins
+Export your data from 1Password as a `.1pif` or `.csv`, depending on which version and operating system you are using:
 
-{% callout info %}
-This set of instructions is only for:
+- `.1pif` files can be exported by **1Password 4 on Windows**, **1Password 6 on macOS**, and **1Password 7 on macOS** (*latest*).
+- `.csv` files can be exported by **1Password 6 on Windows or macOS** and **1Password 7 on Windows or macOS** (*latest*).
 
-- 1Password 4 users on macOS and Windows (only the .agilekeychain format is supported for Windows users)
-- 1Password 6 and 7 users on macOS
+## Export as .1pif
 
-These instructions may vary slightly for macOS users.
-{% endcallout %}
+Complete the following steps to export your 1Password data as a `.1pif` from **1Password 4 on Windows**, **1Password 6 on macOS**, or **1Password 7 on macOS**:
```

Comment:
Do we have reason to think the 1pif format has evolved and we only support these versions? I haven't done any testing with it